lib/SILOptimizer/Transforms/SpecializeOpaqueArchetypes.cpp

```diff
@@ -0,0 +1,466 @@
+#define DEBUG_TYPE "opaque-archetype-specializer"
```

Comment:
This needs a copyright notice, etc

Reply:
https://github.com/apple/swift/pull/24434